code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -210,21 +210,6 @@
 		. = TRUE
 	..()
 
-/datum/reagent/toxin/minttoxin
```

Comment:
needing to repath this is annoying.
toxins do more things then be digested by livers.

it would be nice if ignoring liver modification was a flag toxins can opt into ignoring, rather then something they always do

Reply:
I repathed it under request.  My original fix was to just increase the amount of the reagent.  Would that be preferable?

Adding a flag such as you describe is likely outside my ability.

Reply:
When you have a toxin that has its toxpwr = 0 that's when you kinda need to question whether it even needs to be a toxin at that point.

That said, that `continue` is preventing `on_mob_life` from being called at all if the amount is too low. Which is intentional, and makes sense in general until you want a toxin that does something gimmicky and then try to use `on_mob_life` only to find nothing happens.

And this is not obvious until you trace the code. I think maybe adding a flag there might be a good idea for toxins that you want to always run through `on_mob_life` `on_mob_metabolize` etc. something like a var for `datum/reagent/toxin` called  `always_metabolize` and then check for that alongside the `amount <= liver_tolerance`. 

IMO that should be a separate PR though. This particular chem has no real reason to be a subtype of toxin.

https://github.com/tgstation/tgstation/blob/3a16240e69407d467a6e27e58dcb13dfa4cd11ad/code/modules/reagents/chemistry/holder.dm#L724-L736

Reply:
> needing to repath this is annoying. toxins do more things then be digested by livers.

It's annoying but makes more sense to not be grouped with things that do poison damage like cyanide.  The mint effect is basically an explosion, which is what plenty of the other pyrotechnic reagents do but they don't have their own subtype.  This also means it can be triggered regardless if they have a liver or have more than 3u. 

I have an old branch that moved acid reagents out of the toxin subtype based on the same reasoning.  (should be doing `BURN` damage not TOXIN)
 
> it would be nice if ignoring liver modification was a flag toxins can opt into ignoring, rather then something they always do

Ideally in the future we could get rid of the toxin subtype completely and make it a flag for mob_biotypes to apply to.  Then you could have really interesting effects where some chems are toxic to certain species.  That would be really cool but definitely outside the scope of this PR.

@vinylspiders @the-orange-cow ultimately Lemon gets to make the final call on how he wants this implemented.  I am not a maintainer, however I did rebuild all the liver and toxin code recently so I am well versed in how that mechanic works.

Reply:
I checked, it does hardly anything. Just like plant and mech acid damage.
Only big loss is toxic bees wouldn't spawn with it, but that's ok.

I don't like that this needed to be done, but it does make sense even outside the stupid